src/ui/descendents.rs

```diff
@@ -121,3 +124,11 @@ fn get_proc_children() -> Result<Vec<(pid_t, pid_t)>, Error> {
 
     Ok(pids.iter().map(|&pid| pid as pid_t).zip(ppids).collect())
 }
+
+#[cfg(windows)]
+fn get_proc_children() -> Result<Vec<(pid_t, pid_t)>, Error> {
+    // TODO: this isn't currently implemented, but can be done
+    // using the tlhelp32 Process32First/Process32Next api from the
+    // winapi-rs crate: https://github.com/retep998/winapi-rs/blob/0.3/src/um/tlhelp32.rs#L94
+    Err(format_err!("--subprocesses option is not yet supported on windows"))
```

Comment:
could you change this to `unimplemented!(<msg here>)`, and instead do one of
-  `cfg` out the code that adds the option to the CLI (lhttps://github.com/rbspy/rbspy/blob/111689fe13f843ac6786855ac10ce66c3ad84ab7/src/main.rs#L655)
- fail when matching the command: lhttps://github.com/rbspy/rbspy/blob/111689fe13f843ac6786855ac10ce66c3ad84ab7/src/main.rs#L655

I think it'll be easier to do the second because the args are added in one huuuuuge expression.

Reply:
I went with the second option on this, since the first one would have been a pretty messy change.

I also made the subprocesses option 'hidden' on windows which might help a bit